### PR TITLE
feature: local kubernetes guide

### PR DESCRIPTION
**Pull request by cdxker**
Guide to deploy is mostly finished, tried to reproduce steps on differnet machine and got a 503 from the ingress controller, aside from that it is mostly finishedfrom the ingress controller, aside from that it is mostly finished

### CODE REVIEW

**Comment by cdxker** [comment]
Fixed the issue, the services were only selecting pods that had the `trieve` release being used

File: helm/local-values.yaml [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  clustering_cron:
+    tag: latest
+postgres:
+  useSubchart: true
+  dbURI: postgres://postgres:password@trieve-postgresql.default.svc.cluster.local:5432 # Only used if useSubchart is false
+config:
+  vite:
+    apiHost: http://api.localtrieve.com/api
+    searchUiUrl: http://search.localtrieve.com
+    frontmatterVals: "link,tag_set,time_stamp"
```
Comment (densumesh) [inline helm/local-values.yaml:42]:
```suggestion
```

File: self-hosting.md [hunk 2]
```diff
@@ -185,42 +185,85 @@ We reccomend using `kind`
...
-export S3_SECRET_KEY=ssssssssssssssssssssTTTTTTTTTTTTTTTTTTTT
-export S3_BUCKET=trieve
-export AWS_REGION=us-east-1
-export STRIPE_API_KEY=sk_test_***************************************************************************************************
-export STRIPE_WEBHOOK_SECRET=sk_test_***************************************************************************************************
 
-helm/from-env.sh
+5) Install the helm chart into kubernetes cluster
+
+helm install -f helm/local-values.yaml local helm/
```
Comment (densumesh) [inline self-hosting.md:196]:
```suggestion
`helm install -f helm/local-values.yaml local helm/`
```

**Review by densumesh — DISMISSED** [review]

**Comment by densumesh** [comment]
image

**Comment by cdxker** [comment]
A lot of the other envs and values.yaml need to get changed, for now its just best to have a baseline that we can improve of off

**Review by skeptrunedev — APPROVED** [review]

**Pull request merged**